### PR TITLE
Skip sysctl probe test if no SysV services are found

### PR DESCRIPTION
**Pull request by jan-cerny**
If the list of services returned by `chkconfig --list` is empty,
then the generated OVAL file would be invalid. Since there would
not be anything to test in that situation, it would be better
to skip the test instead. This situation can happen on modern
Linux distributions that have only native systemd services.

### CODE REVIEW

File: tests/probes/runlevel/test_probes_runlevel.sh [hunk 1]
```diff
@@ -25,6 +25,9 @@ function test_probes_runlevel_A {
     require "egrep" || return 255
     require "awk" || return 255
 
+    local services_list=`get_services_list`
```
Comment (cipherboy) [inline tests/probes/runlevel/test_probes_runlevel.sh]: Do you mind using the preferred `local services_list="$(get_services_list)"` syntax? iirc that's what `shellcheck` recommends.

Comment (cipherboy) [reply tests/probes/runlevel/test_probes_runlevel.sh]: (However, the script does use backticks a lot so up to you).

**Comment by cipherboy** [comment]
I haven't validated the functionality of this on a system for which it _is_ empty, but it does not affect a system for which it isn't empty.

File: tests/probes/runlevel/test_probes_runlevel.sh (hunk 1 above)
Comment (jan-cerny) [reply tests/probes/runlevel/test_probes_runlevel.sh]: Yes, I'll change that.

**Comment by mpreisler** [comment]
Shouldn't this be done against `maint-1.2`?

**Comment by jan-cerny** [comment]
Updated to maint-1.2

**Comment by cipherboy** [comment]
Looks good, @jan-cerny, thanks! :)

**Pull request merged**